jekyll/_cci2/faq.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: classic-docs
+title: "Frequently Asked Questions"
+short-title: "FAQ"
+categories: [getting-started]
+---
+
+## Does CircleCI 2.0 run inference commands?
+
+No, CircleCI 2.0 does not infer anything from your project.
+
+This means that no commands are run automatically. Don’t assume that a particular library/package is available unless it is bundled with the Docker image you are using!
+
+## My project is running on CircleCI 2.0, but the build is frozen!
+
+Builds often freeze due to syntax errors in `circle.yml`.
+
+Cancel the build, check your `circle.yml` for proper indents, and ensure that all stages/steps have the required keys.
+
+## Can I use CircleCI 2.0 without creating base images?
+
+Yes, you can use* one of ours!
+
+The `circleci/build-image:ubuntu-14.04-XL-922-9410082` image has the same content as the Ubuntu Trusty 14.04 image our web app uses. Just know that the image is fairly large (around 17.5 GB uncompressed), so it’s less ideal for local testing.
+
+The image defaults to running actions as the `ubuntu` user and is designed to work with network services provided by Docker Compose.
+
+Here’s a [list of languages and tools](https://circleci.com/docs/build-images-2-0/) included in the image.
+
+\*For now. The idea of a monolithic build image doesn’t fit well with the ethos of CircleCI 2.0, so we will eventually deprecate it.
+
+## How do Docker image names work? Where do they come from?
+
+CircleCI 2.0 currently supports pulling (and pushing with Docker Engine) Docker images from [Docker Hub][docker-hub]. For [official images][docker-library], you can pull by simply specifying the name of the image and a tag:
+
+```
+golang:1.7.1
+redis:3.0.7
+```
+
+For public images on Docker Hub, you can pull the image by prefixing the account/team username:
+
+```
+myUsername/couchdb:1.6.1
+```
+
+## Can I use the `latest` tag when specifying image versions?
+
+You _can_, but please don’t. The `latest` tag is mutable and doesn’t always refer to a consistent image SHA. CircleCI runs on a fleet of machines, so a mutable tag could be improperly cached. In that case, re-running a build wouldn’t be deterministic.
```

Comment:
We don't need to do this now, but I think this can be better worded. In the larger Docker/DevOps community, it's common for `latest` to not be the same hash. It's the point and why people use it that way. Not using it on CircleCI is more about reducing variables to make builds more deterministic.